gdx-backend-jtransc/src/com/jtransc/media/limelibgdx/LimeGraphics.java

```diff
@@ -112,6 +123,10 @@ public float getRawDeltaTime() {
 		return deltaTime;
 	}
 
+	@HaxeMethodBody("" +
+		"{% if extra.fps %} return {{ extra.fps }}; {% end %}" +
+		"{% if !extra.fps %} return false; {% end %}"
```

Comment:
This seems to be duplicated or at least to have the same problem. Maybe if this is repeated you can put in a static and call that static inside the body. So you can fix this problem just in one place.